images/win/scripts/Installers/Install-PostgreSQL.ps1

```diff
@@ -0,0 +1,59 @@
+function Install-PostgreSQL {
+    $installPath= choco install postgresql | Select-String "Installed to"
+    return $installPath.Line.split("'")[1]
+}
+
+function Start-PostgreSQL {
+    param(
+        [String]$PostgresPath
+    )
+    $pgbin=Join-path $PostgresPath "bin"
+    $pgdata=Join-path $PostgresPath "data"
+    $startPostgres=Join-path $pgbin "pg_ctl.exe"
+    Start-Process -FilePath $startPostgres -ArgumentList ("-D", "$pgdata", "start")
+    Write-Host "PostgreSQL has been successfully started."
+}
+
+function Ready-PostgreSQL {
```

Comment:
the best practice of PowerShell to start function name with Verb: Validate-PostgreSQL

Reply:
Thank you. Done.